src/plugins/foxPage/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { TrimmedDescriptionLength } from './constants'
+
+export const TrimDescriptionWithEllipsis = (description?: string): string => {
```

Comment:
Util variable name shouldn't be capitalized

Reply:
We already have a few ellipsis utils, could it be added there instead of namespacing this to fox page?
Also, what about making the naming more accurate in that this is an end ellipsis (we already have middle ellipsis utils)?

Reply:
Done ! Moved to `state/slices/portfolioSlice/utils` and renamed to more meaningful name + generalized with trimming length parameter

Reply:
Still not the best place for it to live, but we also have one ellipsis util living in the `components/Modals/FiatRamps` namespace and no global one so that's fine for now.

I'll create a follow-up PR to consolidate these, we don't want specific domains to have general-purpose utils, makes things less maintainable and impacts utils discovery (if you ask yourself "where should I look to find string-related utils?", the answer would probably be in a general purpose `utils/string`)